### DISCUSSION TITLE
How to get the framework of a particular algorithm?

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [biosimulations/biosimulations#4525] [Q&A]
Is there an easy way to determine which modeling framework a particular algorithm/model uses? 

While frameworks are included in the simulator specifications, they are not included in the project descriptions. Projects do contain information about the algorithm used, but I am not sure what the best method to map them to frameworks is.

**Comment by bilalshaikh42** [comment]
@jonrkarr frameworks are actually not specified anywhere in the API.  Is there some mapping between kisao and edam that can be used?

**Comment by jonrkarr** [comment]
There are no links directly among kisao, sbo, and edam.

Kisao ids are in SED-ML files in combine archives. 

The simulator database could be used to create a map from kisao id to sbo id (algorithm to framework).
